pontoon/base/views.py

```diff
@@ -1224,20 +1229,21 @@ def save_to_transifex(request, template=None):
 
 
 @login_required(redirect_field_name='', login_url='/403')
+@require_POST
 @transaction.atomic
-def quality_checks_switch(request):
-    """Turn quality checks on/off for the current user."""
-    log.debug("Turn quality checks on/off for the current user.")
+def toggle_user_profile_attribute(request, email):
+    user = get_object_or_404(User, email=email)
```

Comment:
I didn't catch this. You're right, we should use permission_required_or_403 and definetely test this view to avoid similar problems in future.